### PR TITLE
fixes issue number 36, event pluginn json string with double quotes

### PR DESCRIPTION
**Pull request by aneeshkp**

### CODE REVIEW

**Comment by aneeshkp** [comment]
Duplicate , there is already a PR for this fix https://github.com/redhat-service-assurance/smart-gateway/pull/37

File: internal/pkg/saelastic/utils.go [hunk 1]
```diff
@@ -13,6 +14,39 @@ type EventNotification struct {
 	StartsAt    string
 }
 
+//SanitizeSummary ... summay is description and it adds double quote which breaks the json
+func SanitizeSummary(text string) string {
```
Comment (leifmadsen) [inline internal/pkg/saelastic/utils.go]: I'm not sure I love calling out a specific parameter in the JSON and filtering across it. I have found a particular situation that results in a failed parse, but there are likely other situations that we haven't identified yet that would have a similar situation.

Comment (aneeshkp) [reply internal/pkg/saelastic/utils.go]: @leifmadsen , unfortunately, it has to be string manipulation.  check this playground to see how single escape is also invalid in JSON when it is in name field . Collectd should treat it has a valid JSON string. I will keep looking if there are a better way to fix this. I think the solution I have now should work for all the event-based plugins.
https://play.golang.org/p/PZO3Vy7o6KZ

**Comment by leifmadsen** [comment]
I'm going to close this issue in favour of the solution in #38 only because that's the one I tested, and it seems to resolve the issue.

**Pull request closed without merging**